src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/ProjectsController.java

```diff
@@ -142,6 +140,20 @@ public String getProjectSamplesPage(final Model model, final Principal principal
 		return "projects/project_samples";
 	}
 
+	@RequestMapping(
```

Comment:
This method is also missing a javadoc

Reply:
Strange.  Updated in a3d6355ee4efa543687b4bc398f5f7b550f51fc7